src/glmtools.jl

```diff
@@ -512,7 +525,7 @@ The loglikelihood of a fitted model is the sum of these values over all the obse
 function loglik_obs end
 
 loglik_obs(::Bernoulli, y, μ, wt, ϕ) = wt*logpdf(Bernoulli(μ), y)
-loglik_obs(::Binomial, y, μ, wt, ϕ) = logpdf(Binomial(Int(wt), μ), Int(y*wt))
+loglik_obs(::Binomial, y, μ, wt, ϕ) = logpdf(Binomial(Int(wt), μ), _safe_int(y*wt))
```

Comment:
An alternative would have been
```julia
loglik_obs(::Binomial, y, μ, wt, ϕ) = logpdf(Beta(y*wt + 1, (1 - y)*wt + 1), μ) - log(wt + 1)
```
It would throw when `y*wt` isn't roughly an integer, though.

Reply:
> It would throw when `y*wt` isn't roughly an integer, though.

That's what `_safe_int` does though?

Reply:
We could also just leave out the conversion -- `logpdf(::Binomial)` is defined for nearly integers as well, but returns `-Inf` if you're not near enough, but this would essentially be an untrapped error. I figured it would be better to catch that higher in the call stack so that the error is more interpretable. We could even return more informative ones like "response is not a multiple of weight".